src/cores/stickers/StickerForm.js

```diff
@@ -0,0 +1,229 @@
+import { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateSticker } from '../../features/stickers/stickerSlice'
+import { setAllEvents } from '../../features/events/eventSlice'
+import { useNavigate, useParams, Link } from 'react-router-dom'
+import { saveSticker, editSticker } from '../../services/stickers.services';
+import { fetchAllEvents } from '../../services/events.services';
+import Select from '../../components/select';
+import {fetchAllTeams} from '../../services/team.services'
+import {fetchTeams} from '../../features/teams/teamSlice'
+function StickerForm() {
+
+    const events = useSelector(state => state.events.eventsAll);
+    const eventsOptions = events.map((event) => ({
+        id: event.id,
+        name: event.eventName,
+    }));
```

Comment:
Resumanlo, creen un hook, incluso este hook podria reutilizarse en otros muchisimos sitios
```suggestion
    const eventsOptions = useEventsOptions();
```

Hook en cuestion:
```
function useEventsOptions() {
    const events = useSelector(state => state.events.eventsAll);
    return events.map((event) => ({
        id: event.id,
        name: event.eventName,
    }))
}
```